ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml

```diff
@@ -2728,3 +2728,153 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build03
```

Comment:
[Seems to have lost a race to build03 vs. build01 defaulting][1]:

```
diff -Naupr -I '^[[:space:]]*#.*' /tmp/tmp.DSjPjEJKUA/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-postsubmits.yaml .//ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-postsubmits.yaml
--- /tmp/tmp.DSjPjEJKUA/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-postsubmits.yaml	2023-08-22 22:56:47.[16](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/batch/pull-ci-openshift-release-master-ordered-prow-config/1694121184418861056#1:build-log.txt%3A16)8515346 +0000
+++ .//ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-postsubmits.yaml	2023-08-22 22:56:56.996736419 +0000
@@ -628,7 +628,7 @@ postsubmits:
     always_run: true
     branches:
     - ^main$
-    cluster: build03
+    cluster: build01
     decorate: true
     decoration_config:
       skip_cloning: true
```

Removing this from the merge pool temporarily, but you'll probably need to adjust to catch up to that.

/hold
/test ordered-prow-config

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/batch/pull-ci-openshift-release-master-ordered-prow-config/1694121184418861056

Reply:
@wking So, do we need to rerun `make jobs` to fix this?